### DISCUSSION TITLE
My suggested updated version of the Whitehat Agreement with comments and suggested edits

### DISCUSSION THREAD

**Discussion by Daimon-Law** [security-alliance/safe-harbor#3] [Ideas]
I've done a deep dive on the Whitehat agreement and have redrafted it. Key changes:
- completely redid all numbering in body of agreement so that future edits and additions are simplified.
- moved all common definitions to head of document.
- corrected :-) the formatting for readability.
- Added some critical provisions including in relation to IP, force majeure and confidentiality.
- updated the FAQs to reflect the adjusted terms.

Updated submission can be found on my fork here: https://github.com/Daimon-Law/whitehat-safe-harbor/tree/main/documents 

I can issue a PR if there's value in merging edits.

**Comment by rholterhus** [comment]
Amazing, thank you! I will follow up internally about how we'll integrate the edits and will circle back on this

**Reply by Daimon-Law** [reply]
Thanks @rholterhus - if I could just ask a favour: could you possibly clarify the licence terms for the main repo by adding the appropriate licence? 

Currently the repo has no licence specified so the default position is that the repo is closed source but I assume from the discussion on LeXpunK that these are intended to be open source files that can be used and modified by protocols and their legal teams.

**Reply by rholterhus** [reply]
Yes - we'll figure out the specifics for this and will update. If you (or anyone else reading this) need this clarified within the next week please let me know and I can prioritize, but otherwise will update sometime after March 14 (end of RFC phase). Thanks!
